iothub/device/tests/PlugAndPlay/PnpHelperTests.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Client.PlugAndPlay;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Devices.Client.Tests.PlugAndPlay
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class PnpHelperTests
+    {
+        [TestMethod]
+        public void CreateRootPropertyPatch()
+        {
+            // Format:
+            //  {
+            //      "samplePropertyName": 20
+            //  }
+
+            string propertyName = "someName";
+            int propertyValue = 10;
+
+            string patch = PnpHelper.CreatePropertyPatch(propertyName, JsonConvert.SerializeObject(propertyValue));
+            var jObject = JObject.Parse(patch);
+
+            // There should be a single property added - the above property.
```

Comment:
instead of a comment, how about a failure message instead?

`jobject.Count.Should().Be(1, "there should be a single property added");`